packages/core/src/codewhisperer/service/transformByQ/transformationResultsViewProvider.ts

```diff
@@ -430,6 +430,7 @@ export class ProposedTransformationExplorer {
 
             let deserializeErrorMessage = undefined
             let pathContainingArchive = ''
+            patchFiles = [] // reset patchFiles if there was a previous transformation
```

Comment:
Fun js trick: you can clear an array by setting its length to zero (so the variable could stay `const`, above)

    patchFiles.length = 0